Library/include/CSP/Systems/Users/Authentication.h

```diff
@@ -149,11 +149,12 @@ class CSP_API LoginTokenInfoResult : public ResultBase
 };
 
 
-/// @brief Data structure for an Agora user token, giving userID, channel name and settings regarding sharing of audio/video/screenshare.
+/// @brief Data structure for an Agora user token, giving userID, referenceID, channel name and settings regarding sharing of audio/video/screenshare.
 class CSP_API AgoraUserTokenParams
 {
 public:
 	csp::common::String AgoraUserId;
```

Comment:
I don't actually know what a `ReferenceID` does in this context, could you add a doxygen-style comment explaining it's purpose?

Bonus points for adding similar comments to other member variables whilst you're in the area! 😁

Reply:
Added Doxygen comments for each field